Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardEditViewModel.swift

```diff
@@ -7,35 +7,109 @@ import SwiftUI
 import Common
 import Storage
 
+enum CreditCardEditState: String, Equatable, CaseIterable {
```

Comment:
I think the enums are fine, not sure about the inner enums for the buttons but is understandable. Although IMO they should be part of the view model class instead of declared globally